src/stores/contactStore.ts

```diff
@@ -117,6 +117,14 @@ class ContactStore {
     return this._pbxUsersMap[id]
   }
 
+  getPartyName = async (id: string, callback: Function) => {
```

Comment:
Keep the api calls in a separate file.

Reply:
This is not an API call, It is used to call an API, Thought of writing this in another file as well, but this getPartyName function is related to contactStore scope only. So kept it in this file only.